### PR TITLE
Added Basic Flask App Code in app.py with two routes / and /about

### PR DESCRIPTION
**Pull request by soni0577**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from flask import Flask, render_template
```
Comment (azadatul) [inline app.py]: Add Module doctstring

File: app.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+app = Flask(__name__)
```
Comment (azadatul) [inline app.py]: Please define static and templates folder path explicitly

File: app.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
```
Comment (azadatul) [inline app.py]: define docstring

File: app.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+
+@app.route('/about')
+def about():
+    return render_template('about.html')
```
Comment (azadatul) [inline app.py]: define docstring

File: app.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+from flask import Flask, render_template
+
+app = Flask(__name__)
+
+
+@app.route('/')
```
Comment (rajr25859) [inline app.py]: Home Route ('/') should be accessible by '/home' and '/index'

File: app.py (hunk 1 above)
Comment (soni0577) [reply app.py]: done

File: app.py (hunk 3 above)
Comment (soni0577) [reply app.py]: done

File: app.py (hunk 5 above)
Comment (soni0577) [reply app.py]: done

File: app.py (hunk 4 above)
Comment (soni0577) [reply app.py]: done

File: app.py (hunk 2 above)
Comment (soni0577) [reply app.py]: done

**Comment by rajr25859** [comment]
Looks fine

**Comment by azadatul** [comment]
everything looks fine. Proceed with the merge.

**Pull request merged**